lib/sentry/plug_context.ex

```diff
@@ -84,7 +84,29 @@ defmodule Sentry.PlugContext do
 
   For example:
 
-      plug Sentry.PlugContext, cookie_scrubber: &MyCookieScrubber.scrub_cookies/1
+      plug Sentry.PlugContext, cookie_scrubber: &MySentryScrubber.scrub_cookies/1
+
+  ### Scrubbing URLs
```

Comment:
We need to say that this has not always been available

```suggestion
  ### Scrubbing URLs

  *Available since v10.2.0.*
```

Reply:
Done! Thanks for the `suggestion`.